packages/core/src/components/fields/SchemaField.js

```diff
@@ -68,10 +68,7 @@ function Label(props) {
     return null;
   }
   return (
-    <label
-      className="control-label"
-      htmlFor={id}
-      aria-describedby={ariaDescribedBy(id)}>
+    <label className="control-label" htmlFor={id}>
```

Comment:
Why was the original aria-describedby here incorrect?

Reply:
I added it by mistake here: https://github.com/rjsf-team/react-jsonschema-form/pull/2386/commits/d332de122ed74dd534334d0c2835a23ce1577811

Original is correct: `<input>` has `id`, `<label>` has `for`. `aria-describedby` goes also to the `<input>`